lisa/tools/parted.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Parted(Tool):
```

Comment:
What's the difference with fdisk? Do we need both of them?

Reply:
parted can partition disk bigger than 2 TB

Reply:
So, add comments in the tool.

Reply:
fixed